orders/OrderBot.ts

```diff
@@ -42,7 +42,25 @@ export class OrderBot extends BaseSushiBot {
     }
 
     private getPropertyEmbed(data: any, propertyName: string): APIEmbedField | null {
-        const propertyValue = this.getTextPropertyValue(data, propertyName);
+        var propertyValue;
```

Comment:
nit: use `let` instead
```suggestion
        let propertyValue;
```